tutorial/testing.py

```diff
@@ -182,6 +182,66 @@ def test_bsly001_falsy_child(dash_duo):
     variable, so the regression result is only available in Plotly's CircleCI
     setup.
 
+    ## Dash for R testing
+
+    We released [Dash for R](https://medium.com/@plotlygraphs/announcing-dash-for-r-82dce99bae13)
+    in July 2019.  To facilitate testing it, we extended the *Python* package
+    `dash.testing` to support Dash for R apps.
+
+    The new `dashr` fixture gives us a test development experience in
+    Dash for R that's nearly identical to the `dash_duo` fixture in
+    Dash for Python. In this context, a `dashr` fixture is an instance of
+    Python class which provides a reliable baseline to execute the app and test
```

Comment:
```suggestion
    Python class which provides a reliable framework to execute the app and test
```
"Baseline" to me means some complete state that you add a particular perturbation to, but `dashr` isn't complete on its own, it's more of a structure into which you insert your test app.